internal/analytics/client.go

```diff
@@ -18,9 +18,20 @@ type Tracker interface {
 		eventName string,
 		properties map[string]interface{},
 	)
+	SendCommandRunEvent(
```

Comment:
We don't need `SendEvent` in the interface now, and it could be an unexported method in the implementations.